src/plugins/graph/components/graph.tsx

```diff
@@ -102,14 +104,18 @@ export const Graph = observer(
     const disposer = graphModel && onAnyAction(graphModel, action => {
       if (isAttributeAssignmentAction(action)) {
         let graphPlace: GraphPlace = "yPlus";
-        let dataSetId = dataset?.id ?? "";
+        let dataSetId = dataset?.id ?? ""; // FIXME for multi-dataset
         let attrId = "";
         if (isSetAttributeIDAction(action)) {
           const [role, _dataSetId, _attrId] = action.args;
           graphPlace = attrRoleToGraphPlace[role] as GraphPlace;
           dataSetId = _dataSetId;
           attrId = _attrId;
         }
+        else if (isRemoveAttributeAction(action)) {
```

Comment:
I think it would make sense to have a comment here about the fact that this is a `DataSet` action, given that `onAnyAction` is called on the graph model and the other methods listened to here are graph model actions.